_config/site.yml

```diff
@@ -98,6 +98,7 @@ profiles:
     js_minifier: disabled
     html_minifier: disabled
     legacy_post_syntax_highlighting: disabled
+    ignore_older_than_days: 30
```

Comment:
By the way, would it be possible to change it so that it keeps the latest 30 posts?
I would like to be able to specify the number of posts instead of the date.